k8s/local.yaml

```diff
@@ -0,0 +1,521 @@
+# pachctl deploy local --dry-run -o yaml
```

Comment:
Do you see any way we could generate this rather than checking it in (maybe generate it and store it in a tmp directory?). Maybe it's necessary, but I just ask because offhand this seems liable to get out of date.

Reply:
yeah, I'm concerned about it getting out of date as well, in the interim, we could have people just generate manually